lib/rubocop/options.rb

```diff
@@ -14,6 +14,9 @@ module OptionsHelp
       config:               'Specify configuration file.',
       auto_gen_config:     ['Generate a configuration file acting as a',
                             'TODO list.'],
+      exclude_limit:       ['Used together with --auto-gen-config to',
+                            'set the limit for how many Exclude',
+                            'properties to generate. Default is 15.'],
```

Comment:
This is not so good, that 15 appears in two places. Seems better to have a constant, and to put it in this file.